samples/wifi/shell/prj_ipv6only.conf

```diff
@@ -0,0 +1,79 @@
+#
+# Copyright (c) 2022 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+# Configuration for IPv6-only build variant of the shell sample.
+# It relies on Router Advertisements rather than DHCP or static
+# configuration.
+
+# Wi-Fi driver
```

Comment:
Can we consider adding `CONFIG_NRF_WIFI_LOW_POWER` here as well? We've had issues with this option enabled in Matter samples.

Reply:
We can, but it doesn't really work for me, so I wasn't sure we would like to provide an unstable config until it is investigated and fixed. If you think we would, I can add it.

Reply:
It is being tested, when it works it will be enabled in the main `prj.conf`

Reply:
Ok, then let's just wait.